test/spec/modules/jwplayerBidAdapter_spec.js

```diff
@@ -37,7 +45,62 @@ describe('jwplayer adapter tests', function() {
     });
   });
 
-  describe('buildRequests for video', function() {});
+  describe('buildRequests for video', function() {
+    it('buildRequests works', function() {
+      const bidRequests = [
+        {
+          'bidder': 'jwplayer',
+          'params': {
+            'placementId': 12345
+          },
+          'mediaTypes': {
+            'video': {
+              'playerSize': [640, 480],
+              'content': {}
+            }
+          },
+          'bidRequestsCount': 1,
+          'adUnitCode': 'testAdUnitCode',
+          'bidId': 'testBidId'
+        }
+      ]
+
+      const serverRequests = spec.buildRequests(bidRequests, this.defaultBidderRequest);
+
+      /* eslint-disable no-console */
+
+      serverRequests.forEach(serverRequest => {
+        expect(serverRequest.url).to.have.string('https://ib.adnxs.com/openrtb2/prebid');
+        expect(serverRequest.method).to.equal('POST');
+
+        const openrtbRequest = JSON.parse(serverRequest.data);
+
+        expect(openrtbRequest.id).to.not.equal(null);
+        expect(openrtbRequest.id).to.have.string('testBidId');
+
+        expect(openrtbRequest.site).to.not.equal(null);
+        expect(openrtbRequest.site).to.be.an('object');
```

Comment:
We should add a test for site checking that
- config.ortb2.site gets used
- site.page
- site.domain
- site.ref

Reply:
added

Reply:
done.